tests/test_lr_finder.py

```diff
@@ -4,6 +4,14 @@
 import task as mod_task
 
 
+try:
+    import apex
```

Comment:
I think this line can be changed to `from apex import amp` and we can then remove the local imports from the functions

Reply:
Yeah, I'll fix it.